distribution/docs/src/main/resources/_contents/_data/contact-attributes-table-contents.adoc

```diff
@@ -0,0 +1,109 @@
+
+.[[_contact_attributes_table]]Contact: Attributes in this group reflect metadata about different kinds of people/groups/units/organizations that can be associated with a metacard.
+[cols="1,2,1,1,1" options="header"]
+|===
+|Term
+|Definition
+|Datatype
+|Constraints
+|Example Value
+
+|contact.creator-name
+|The name of this metacard’s creator.
```

Comment:
It's just a string, so functionally it makes little difference. We should still establish a convention. Given that these names are displayed verbatim in the UI, I favor something human-readable like `titles prefixes first middle last suffixes` though obviously some of those are optional.